doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='clustering-datagrid-proc_{context}']
+= Installing and configuring {DATAGRID}
+
+To use {DATAGRID} for more efficient searching across cluster nodes, you must install and configure {DATAGRID} for the {PRODUCT} clustered environment. Use the following instructions to configure a simplified, non-high availability environment.
```

Comment:
"you must" is not required in this sentence (minimalism)
:To use {DATAGRID} for more efficient searching across cluster nodes, install and configure {DATAGRID} for the {PRODUCT} clustered environment.

Reply:
Done